lib/net/pool.js

```diff
@@ -597,9 +628,9 @@ class Pool extends EventEmitter {
 
     const host = IP.toHostname(ip, socket.remotePort);
 
-    assert(!this.peers.map.has(host), 'Port collision.');
+    assert(!this.peers.map.has(host), 'Port collision (clear).');
```

Comment:
Nit: no need to have `clear` here, this is an artifact of when two peers lists were being maintained. I decided it would be more simple to just maintain a single peers list.